### DISCUSSION TITLE
Why delete endpoint name query keyword from oap-server ?

### DISCUSSION THREAD

**Discussion by andotorg** [apache/skywalking#9120] [Q&A] [answered]
I think oap-server should keep the endpoint name querykeyword

**Comment by wu-sheng** [comment, marked as answer]
Why again? I think we jave discussed this.

**Reply by andotorg** [reply]
No, last time we discussed why the UI doesn't exist. I think we can let the server save the query field? This will provide compatibility support for older versions of the UI

**Reply by wu-sheng** [reply]
No, these are the same thing. We never declared our support for older versions. This is a 5 years tradition. Same as last one, we don't have people willing to support older ones.

**Reply by andotorg** [reply]
ok
